os-boot/opensbi/sail.dts

```diff
@@ -5,6 +5,9 @@
   #size-cells = <2>;
   compatible = "ucbbar,spike-bare-dev";
   model = "ucbbar,spike-bare";
+  chosen {
+    bootargs = "console=hvc0 earlycon=sbi";
```

Comment:
These are Linux-specific, do they really belong here?

Reply:
That doesn't mater I think(